src/main/java/hudson/tasks/test/TestObject.java

```diff
@@ -189,15 +204,15 @@ public String getRelativePathFrom(TestObject it) {
                 buf.insert(0, Functions.getRelativeLinkTo(myBuildAsItem));
             } else {
                 // We're not in a stapler request. Okay, give up.
-                LOGGER.fine("trying to get relative path, but it is not my ancestor, and we are not in a stapler request. Trying absolute hudson url...");
-                String hudsonRootUrl = Helper.getActiveInstance().getRootUrl();
-                if (hudsonRootUrl==null||hudsonRootUrl.length()==0) {
+                LOGGER.fine(() -> "trying to get relative path, but it is not my ancestor, and we are not in a stapler request. Trying absolute jenkins url...");
```

Comment:
What is the point of lazily evaluating a string literal?

Reply:
Fixed it didn't realize it was just a pure string literal